### ISSUE TITLE
File/String usage in documentation

### ISSUE THREAD

**Issue by I3ck** [bmwill/diffy#3]
Within the documentation most parameters are described as `file`. But it seems like the parameters are actually `strings` (at least in many cases).  
I think it makes sense to update the documentation accordingly.

**Comment by bmwill** [comment]
Yes I can see how that would be slightly confusing. The intent of the library is to provide the functionality for diffing/merging files but in order to be more flexible all of the interfaces work on borrowed strings instead of a `std::fs::File` type. The expected usage would be to first read a file into a String and then use this library. I'll try and make suitable adjustments to the documentation to reflect this

**Comment by bmwill** [comment]
Latest release has some updated docs explaining this.
